seatunnel-connectors-v2/README.md

```diff
@@ -79,6 +80,6 @@ It is used to process ``SinkWriter.prepareCommit`` the returned data information
 In the current version, it is recommended to implement ``SinkAggregatedCommitter`` as the first choice, which can provide strong consistency guarantee in Flink/Spark. At the same time, commit should be idempotent, and save engine retry can work normally.
```

Comment:
SeaTunnel has designed a new connector API to decouple from the compute engine. This article introduces the new interface and the new code structure to help developers quickly get started with the new API and understand the principles of the new API.